src/resources/views/order/show/_update_billpayer_modal.blade.php

```diff
@@ -0,0 +1,213 @@
+
+                    <div class="col-md-6">
+                        <label for="postalcode" class="form-label">{{ __('Postal Code') }}</label>
+
+                        <div class="mb-3">
+                            {{ Form::text('billpayer[address][postalcode]', old('billpayer.address.postalcode', $billpayerAddress->postalcode), ['class' => 'form-control' . ($errors->has('billpayer.address.postalcode') ? ' is-invalid' : ''), 'id' => 'postalcode']) }}
+
+                            @if ($errors->has('billpayer.address.postalcode'))
+                                <div class="invalid-feedback">{{ $errors->first('billpayer.address.postalcode') }}</div>
+                            @endif
+                        </div>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col-md-6">
+                        <label for="city" class="form-label">{{ __('City') }}</label>
+
+                        <div class="mb-3">
+                            {{ Form::text('billpayer[address][city]', old('billpayer.address.city', $billpayerAddress->city), ['class' => 'form-control' . ($errors->has('billpayer.address.city') ? ' is-invalid' : ''), 'id' => 'city']) }}
+
+                            @if ($errors->has('billpayer.address.city'))
+                                <div class="invalid-feedback">{{ $errors->first('billpayer.address.city') }}</div>
+                            @endif
+                        </div>
+                    </div>
+
+                    <div class="col-md-6">
+                        <label for="address" class="form-label">{{ __('Address') }}</label>
+
+                        <div class="mb-3">
+                            {{ Form::text('billpayer[address][address]', old('billpayer.address.address', $billpayerAddress->address), ['class' => 'form-control' . ($errors->has('billpayer.address.address') ? ' is-invalid' : ''), 'id' => 'address']) }}
+
+                            @if ($errors->has('billpayer.address.address'))
+                                <div class="invalid-feedback">{{ $errors->first('billpayer.address.address') }}</div>
+                            @endif
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="modal-footer">
+                <button type="button" class="btn btn-link" data-bs-dismiss="modal">{{ __('Close') }}</button>
+                <button class="btn btn-primary">{{ __('Save') }}</button>
+            </div>
+
+            {!! Form::close() !!}
+        </div>
+    </div>
+</div>
+
+@push('scripts')
+    <script>
+        document.addEventListener('alpine:init', () => {
+            Alpine.data('editBillpayerModal', () => ({
+                isOrganization: @json(old('is_organization') === null && $billpayer->is_organization) || {{ old('billpayer.is_organization') == '1' ? 'true' : 'false' }} || @json(!empty($billpayer->company_name)),
```

Comment:
I'm not 100% sure TBH, but I think this is a better condition
```suggestion
                isOrganization: @json(old('billpayer.is_organization') == '1' || !empty($billpayer->company_name)),
```